apps/mobile-wallet/src/screens/NFTs/NFTListScreen.tsx

```diff
@@ -19,35 +19,27 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 import { StackScreenProps } from '@react-navigation/stack'
 import { useTranslation } from 'react-i18next'
 
-import BottomBarScrollScreen, { BottomBarScrollScreenProps } from '~/components/layout/BottomBarScrollScreen'
+import BaseHeader from '~/components/headers/BaseHeader'
+import Screen, { ScreenProps } from '~/components/layout/Screen'
+import ScreenTitle from '~/components/layout/ScreenTitle'
 import NFTsGrid from '~/components/NFTsGrid'
+import useScreenScrollHandler from '~/hooks/layout/useScreenScrollHandler'
 import { InWalletTabsParamList } from '~/navigation/InWalletNavigation'
-import { ReceiveNavigationParamList } from '~/navigation/ReceiveNavigation'
-import { SendNavigationParamList } from '~/navigation/SendNavigation'
 
-interface ScreenProps
-  extends StackScreenProps<
-      InWalletTabsParamList & ReceiveNavigationParamList & SendNavigationParamList,
-      'NFTListScreen'
-    >,
-    BottomBarScrollScreenProps {}
+interface NFTListScreenProps extends StackScreenProps<InWalletTabsParamList, 'NFTListScreen'>, ScreenProps {}
```

Comment:
nit: could use a type here to avoid the additional `{}`

Reply:
Done